### PR TITLE
[MODULAR] More Wings

### PR DESCRIPTION
**Pull request by Nerev4r**
## About The Pull Request

<details>
<summary> Wings (still can't fly lol) </summary>

image
image
image
_Insectoid_ wings from _Hyper._ *chitter.

image
image
image
_Beetle_ wings from _Hyper._ Also *chitter.

image
image
image
_Mini-Feathery_ wings from _Hyper._ These are just adorable. It's a shame they don't fit on Teshari.

image
image
image
_Tiny-Feathery_ wings from _Hyper._ These are somehow even more adorable. My heart.

image
image
image
_Mini-Bat_ wings from _Hyper._ For short succubi.

image
image
image
_Tiny-Bat_ wings from _Hyper._ For those characters that only really *squeak.
</details>

## How This Contributes To The Skyrat Roleplay Experience
Adds new customization options borrowed from https://github.com/quotefox/Hyper-Station-13, https://github.com/VOREStation/VOREStation/pulls, https://github.com/Citadel-Station-13/Citadel-Station-13-RP, and https://github.com/sojourn-13/sojourn-station.

## Changelog

:cl:
imageadd: More wings, about eight of them. Miniature (and tinier) bat and feathery wings, as well as beetle and generic insectoid wings.
/:cl:

### CODE REVIEW

**Review by tf-4 — APPROVED** [review]
The mantis arms look kinda strange but I assume they're not really intended for humans anyway. Code looks fine. Rest of the sprites are nice.

File: modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/wings.dm [hunk 1]
```diff
@@ -295,3 +295,52 @@
 	icon_state = "robowing"
 	color_src = USE_MATRIXED_COLORS
 
+/datum/sprite_accessory/wings/mammal/dragonfly
+	name = "Dragonfly"
+	icon_state = "dragonfly"
+	color_src = USE_ONE_COLOR
+
+/datum/sprite_accessory/wings/mammal/beealt
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/wings.dm]:
```suggestion
/datum/sprite_accessory/wings/mammal/bee_alt
```

File: modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/wings.dm [hunk 2]
```diff
@@ -295,3 +295,52 @@
...
+	name = "Mantis Arms"
+	icon_state = "mantisarms"
+	color_src = USE_MATRIXED_COLORS
+
+/datum/sprite_accessory/wings/mammal/top/mantisarms
+	name = "Mantis Arms (Top)"
+	icon_state = "mantisarms"
+	color_src = USE_MATRIXED_COLORS
+
+/datum/sprite_accessory/wings/mammal/minibat
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/wings.dm]:
```suggestion
/datum/sprite_accessory/wings/mammal/mini_bat
```

File: modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/wings.dm [hunk 3]
```diff
@@ -295,3 +295,52 @@
...
+	name = "Mini-Bat"
+	icon_state = "minibat"
+	color_src = USE_ONE_COLOR
+
+/datum/sprite_accessory/wings/mammal/minifeather
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/wings.dm]:
```suggestion
/datum/sprite_accessory/wings/mammal/mini_feather
```

File: modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/wings.dm [hunk 4]
```diff
@@ -295,3 +295,52 @@
...
+	name = "Mini-Feathery"
+	icon_state = "minifeather"
+	color_src = USE_ONE_COLOR
+
+/datum/sprite_accessory/wings/mammal/tinybat
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/wings.dm]:
```suggestion
/datum/sprite_accessory/wings/mammal/tiny_bat
```

File: modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/wings.dm [hunk 5]
```diff
@@ -295,3 +295,52 @@
...
+	name = "Tiny-Bat"
+	icon_state = "tinybat"
+	color_src = USE_ONE_COLOR
+
+/datum/sprite_accessory/wings/mammal/tinyfeather
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/wings.dm]:
```suggestion
/datum/sprite_accessory/wings/mammal/tiny_feather
```

File: modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/wings.dm [hunk 6]
```diff
@@ -295,3 +295,52 @@
 	icon_state = "robowing"
 	color_src = USE_MATRIXED_COLORS
 
+/datum/sprite_accessory/wings/mammal/dragonfly
+	name = "Dragonfly"
+	icon_state = "dragonfly"
+	color_src = USE_ONE_COLOR
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/wings.dm]: You haven't shown sprite pictures of this one.

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
I really like the last four, not sure about the alt bee one, the rest are fine, except the mantis one.

As I said in the taur PR, I'm not a big fan of adding "wings" that both don't behave like wings, and make it look as if you've got a second pair of arms, even if you don't, which is kind of misleading and... Visually incoherent, so to speak? I'm just really not convinced, honestly.

**Review by GoldenAlpharex — APPROVED** [review]
I really like the tiny and mini ones, they're very cute :)

**Pull request merged**